python/pyspark/ml/tests.py

```diff
@@ -1027,6 +1027,21 @@ def test_storage_levels(self):
         self.assertEqual(als.getFinalStorageLevel(), "DISK_ONLY")
         self.assertEqual(als._java_obj.getFinalStorageLevel(), "DISK_ONLY")
 
+    def test_unknown_strategy(self):
```

Comment:
I don't think we need to test this at all. We already have tests to check the param setters and getters and transfer to java, and that's really all this does. I don't know why we would need to check this, but _not_ check the many other params ALS has.